common/awf_vehicle_rviz_plugin/awf_2d_overlay_vehicle/awf_2d_overlay_vehicle-extras.cmake

```diff
@@ -0,0 +1,30 @@
+# Copyright (c) 2021, Open Source Robotics Foundation, Inc.
```

Comment:
https://github.com/autowarefoundation/autoware.universe/actions/runs/7653788429/job/20856395515#step:7:314

It seems the license type is not declared here, let me look deeper.

Reply:
Found the source:

https://github.com/ros2/rviz/blob/rolling/rviz_default_plugins/rviz_default_plugins-extras.cmake

Reply:
RViz2 has "The Clear BSD License" it seems.

https://github.com/ros2/rviz/blob/rolling/LICENSE

Reply:
```suggestion
# Copyright (c) 2021, Open Source Robotics Foundation, Inc.
# The Clear BSD License
```

Reply:
It seems that the copyright notice must exactly match [this file](https://github.com/ament/ament_lint/blob/humble/ament_copyright/ament_copyright/template/bsd_3clause_license.txt). I confirmed that the following changes pass the ament_copyright check.
- Undo adding `The Clear BSD License`
- Move `All rights reserved.` to the same line as `Copyright (c) ...`
- Replace `the Willow Garage, Inc.` with `the copyright holder`
    - I think this needs to be moved to the top `copyright (c)` part, but I don't know the publication year.
- Replace `THE COPYRIGHT OWNER` with `THE COPYRIGHT HOLDER`

My opinion is to add the find_package for qt to CMakeLists.txt and delete this file. Adding find_package is a common practice so I don't think it's necessary to keep this file.